### ISSUE TITLE
Possibility of choosing which folders to display images from in the gallery view 

### ISSUE THREAD

**Issue by Darmstadtio** [Infomaniak/android-kDrive#1476]
As per title, it would be very convenient to be able to choose to display in the gallery view only images contained in certain folders, selectable by the user.

At the moment it's very frustrating as it's not possible to separate images from the actual camera from those that are various, generic, ‘for other’ uploads.

Is a generic archive. At this moment the gallery view is almost (sadly) pointless.

**Comment by marco-dimeo** [comment]
Good idea! I'm trying kdrive and I've the same think.

**Comment by sirambd** [comment]
> As per title, it would be very convenient to be able to choose to display in the gallery view only images contained in certain folders, selectable by the user.
> 
> At the moment it's very frustrating as it's not possible to separate images from the actual camera from those that are various, generic, ‘for other’ uploads.
> 
> Is a generic archive. At this moment the gallery view is almost (sadly) pointless.

@Darmstadtio Thanks for your suggestion! We understand how filtering images by folder would make the gallery view much more intuitive and useful.

We’re always looking to improve the experience, and this is definitely an idea worth considering. To help us track interest and prioritize features, we encourage you to share (or upvote) this request on our UserReport feedback forum: [Submit or Vote Here](https://feedback.userreport.com/1d484922-97e1-4a43-a4c9-4d19bc1b5494). This allows us to gauge community demand and make informed decisions.

We really appreciate your input—keep the great ideas coming! 🚀

**Comment by sirambd** [comment]
> Good idea! I'm trying kdrive and I've the same think.

@marco-dimeo You can go there to vote the feedback

**Comment by marco-dimeo** [comment]
I've inserted the request in the feedback forum 😉

**Comment by hugo-ym** [comment]
Good idea. Perhaps just add a status for each folder with 2 possibilities: include/exclude from gallery.

**Comment by Lnamw** [comment]
Issue cleanup 🧹
